### ISSUE TITLE
Why do the results from NRE seem to perform better?

### ISSUE THREAD

**Issue by lanzhuzhu** [Panda0406/Adversarial-Learning-Distant-Supervision-RE#2]
Thank you for your open sourcing. Your code is really clear. 

I removed the original figure for making a mistake about the original results.

**Comment by Panda0406** [comment]
Thank you for your attention.

First, for the results in [thunlp/NRE](https://github.com/thunlp/NRE)  is on the dataset that has 570088 sentences. This dataset has some repetitive sentences both in the training set and the test set. They have corrected it, and the new dataset has 522611 sentences. This new dataset has low performance.

Second, the **original PCNN+ATT results** presented in my repository is run by myself. I cannot get the result as they perform. And the results presented in my repository is similar to the results presented in their [paper](https://www.aclweb.org/anthology/P16-1200).

**Comment by Panda0406** [comment]
We just generate a cleaned dataset, and use their thunlp/NRE code to run on this new dataset to get our results.

**Comment by Panda0406** [comment]
Please show me "copyfromNRE/PCNN+ATT.txt" file. I have tried the pr.txt file from [thnlp/NRE](https://github.com/thunlp/NRE), but it is not like the curve you present.

**Comment by lanzhuzhu** [comment]
> Please show me "copyfromNRE/PCNN+ATT.txt" file. I have tried the pr.txt file from [thnlp/NRE](https://github.com/thunlp/NRE), but it is not like the curve you present.

Thank you very much for your prompt reply, the pr.txt I used is from [their repo](https://github.com/thunlp/NRE/blob/master/PCNN%2BATT/out/pr.txt)

**Comment by lanzhuzhu** [comment]
Another question is why the clean data generated 

> We just generate a cleaned dataset, and use their thunlp/NRE code to run on this new dataset to get our results.

Another question, when I ran your code and generated the clean data, I found there are more sentences than the original training data. The clean data is selected from the original training data, why it has more sentences?

**Comment by Panda0406** [comment]
[Their repo](https://github.com/thunlp/NRE/blob/master/PCNN%2BATT/out/pr.txt) do not perform as you present. I have tested it.

Because we remove many sentence bags of positive entity pairs, it causes the proportion change of positive set and negative set. So we correspondingly add some positive sentence sets based on the learned filter network. These positive sentence sets are the subset of cleaned positive sentence sets. This part is explained in the code.

**Comment by lanzhuzhu** [comment]
> [Their repo](https://github.com/thunlp/NRE/blob/master/PCNN%2BATT/out/pr.txt) do not perform as you present. I have tested it.
> 
> Because we remove many sentence bags of positive entity pairs, it causes the proportion change of positive set and negative set. So we correspondingly add some positive sentence sets based on the learned filter network. These positive sentence sets are the subset of cleaned positive sentence sets. This part is explained in the code.

Thank you for your reply.  I re-downloaded the PCNN+ATT.txt result file, and it presented differently from the previous figure. I am sorry for making such a fault.  Maybe you can delete this issue in case of making others misunderstand.
